contracts/RiskFund/RiskFund.sol

```diff
@@ -142,6 +143,58 @@ contract RiskFund is
         emit MinAmountToConvertUpdated(oldMinAmountToConvert, minAmountToConvert_);
     }
 
+    /**
+     * @notice Sets a new convertible base asset and swaps previous base asset to new base asset
+     * @param _convertibleBaseAssetVToken Address for new convertible base asset.
+     * @param markets Markets to swap underlying base asset.
+     * @param amountsOutMin Minimum amount to swap
+     * @param paths Path to swap between old and new convertible base asset.
+     */
+    function setConvertibleBaseAsset(
+        address _convertibleBaseAssetVToken,
+        address[] calldata markets,
+        uint256[] calldata amountsOutMin,
+        address[][] calldata paths
+    ) external onlyOwner {
+        require(_convertibleBaseAssetVToken != address(0), "Risk Fund: new convertible base asset address invalid");
+        require(markets.length == amountsOutMin.length, "Risk fund: markets and amountsOutMin are unequal lengths");
+
+        VToken vToken = VToken(_convertibleBaseAssetVToken);
```

Comment:
I talked briefly with Narayan about this, I think we do need this constraint to make sure we have oracle support for the convertible base asset. Otherwise we need to plan how to support it with an oracle